modules/odc-installing-helm-charts.adoc

```diff
@@ -19,15 +19,17 @@ image::odc_helm_chart_devcatalog.png[]
 +
 . In the *Install Helm Chart* page:
 .. Enter a unique name for the release in the *Release Name* field.
-.. Select the required chart version from the *Chart Version* dorp-down.
+.. Select the required chart version from the *Chart Version* drop-down list.
 +
 .Select chart version
 image::odc_helm_chart_select_chart_ver.png[]
 +
-.. You can use the *Form view* of the *YAML view* to configure your Helm chart.
-.. Click *Install* to save the required changes and install the chart.
+.. Use the *Form view* or the *YAML view* to configure your Helm chart.
```

Comment:
Please add the note on lines 30 and 31 below this bullet point. That information will be of no use to the users once he clicks install. He will be able to switch only while configuring. Add the content in the place where it is most relevant to the users and where the information to the user is most useful.

Reply:
Done, incorporated the change. Please check.